CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Features
 ### Enhancements
 ### Bug Fixes
+* Fixed jni test and compile error ([#940](https://github.com/opensearch-project/k-NN/pull/940))
```

Comment:
can we put this like `Fixed jni test and compile warnings`, as if it would have been error the build should have faied.

Reply:
++